operator/internal/central/values/translation/translation.go

```diff
@@ -172,23 +178,38 @@ func getExposureRouteValues(r *platform.ExposureRoute) *translation.ValuesBuilde
 	return &route
 }
 
-func getCentralDBPersistenceValues(p *platform.DBPersistence) *translation.ValuesBuilder {
+func getCentralDBPersistenceValues(ctx context.Context, p *platform.DBPersistence, namespace string, client ctrlClient.Client) *translation.ValuesBuilder {
 	persistence := translation.NewValuesBuilder()
 	if hostPath := p.GetHostPath(); hostPath != "" {
 		persistence.SetStringValue("hostPath", hostPath)
 	} else {
 		pvcBuilder := translation.NewValuesBuilder()
 		pvcBuilder.SetBoolValue("createClaim", false)
+
+		// Search for a backup PVC, if it exists, allow to mount it.
+		claimName := common.DefaultCentralDBBackupPVCName
 		if pvc := p.GetPersistentVolumeClaim(); pvc != nil {
+			claimName = common.GetBackupClaimName(*pvc.ClaimName)
 			pvcBuilder.SetString("claimName", pvc.ClaimName)
 		}
 
+		key := ctrlClient.ObjectKey{
+			Namespace: namespace,
+			Name:      claimName,
+		}
+		pvc := &corev1.PersistentVolumeClaim{}
+		if client.Get(ctx, key, pvc) == nil {
+			// If faced error of any kind (object not found or a transient
+			// error), assume we're not able to mount a backup volume.
+			persistence.SetBoolValue("backup", true)
```

Comment:
I'm wondering, if we're not advertising this to the users, should this rather be something like:
```suggestion
			persistence.SetBoolValue("_backup", true)
```